src/internet_identity/src/migrations/lookup_anchor_with_recovery_phrase_principal.rs

```diff
@@ -0,0 +1,125 @@
+use ic_stable_structures::Memory;
+
+use crate::{
+    storage::Storage, RECOVERY_PHRASE_MIGRATION_BATCH_ID, RECOVERY_PHRASE_MIGRATION_BATCH_SIZE,
+    RECOVERY_PHRASE_MIGRATION_ERRORS, RECOVERY_PHRASE_MIGRATION_LAST_ANCHOR_ID,
+};
+
+impl<M: Memory + Clone> Storage<M> {
+    /// Migrates the next batch of anchors to synchronize the following indices:
+    /// `lookup_anchor_with_recovery_phrase_principal_memory_wrapper`
```

Comment:
Is it `lookup_anchor_with_recovery_phrase_principal_memory_wrapper` or just `lookup_anchor_with_recovery_phrase_principal`?